SecondTriggerNew.trigger

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------------------------
+
+Q2.System.debug the following statements for all events(Before and after)
+a)Trigger.new b)Trigger.old c)Trigger.newMap d)Trigger.oldmap
+
+--------------------------------------------------------------------------------------------------------------*/
+
+trigger SecondTriggerNew on Contact (after insert, after update, before insert, before update) {
+    TriggerSetting__c triggerSettingInstance =  TriggerSetting__c.getInstance();
+	if(triggerSettingInstance.ContactObject__c){
```

Comment:
This is hardcode dependency on Custom setting field which later leads to be create more fields for each trigger/object. Which is not fesible solution.